server/applicationVersion.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable import/no-unresolved,global-require
```

Comment:
moved these functions out to align with [how we do it in the template project](https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/applicationVersion.ts)